hip/README.md

```diff
@@ -0,0 +1,130 @@
+               __                __
+              / /   ____  ____  / /_  ____  _____
+             / /   / __ `/ __ `/ __ \/ __ \/ ___/
+            / /___/ /_/ / /_/ / / / / /_/ (__  )
+           /_____/\__,_/\__, /_/ /_/\____/____/
+                       /____/
+
+        High-order Lagrangian Hydrodynamics Miniapp
+
+                      HIP version
+
+## Overview
+
+This directory contains the HIP version of the **Laghos** (LAGrangian
+High-Order Solver), which is provided as a reference implementation and is NOT
+the official benchmark version of the miniapp.
+
+For more details about Laghos see the [README file](../README.md) in the
+top-level directory.
+
+The Laghos miniapp is part of the [CEED software suite](http://ceed.exascaleproject.org/software),
+a collection of software benchmarks, miniapps, libraries and APIs for
+efficient exascale discretizations based on high-order finite element
+and spectral element methods. See http://github.com/ceed for more
+information and source code availability.
+
+The CEED research is supported by the [Exascale Computing Project](https://exascaleproject.org/exascale-computing-project)
+(17-SC-20-SC), a collaborative effort of two U.S. Department of Energy
+organizations (Office of Science and the National Nuclear Security
+Administration) responsible for the planning and preparation of a
+[capable exascale ecosystem](https://exascaleproject.org/what-is-exascale),
+including software, applications, hardware, advanced system engineering and early
+testbed platforms, in support of the nation’s exascale computing imperative.
+
+## Differences with the official benchmark version
+
+The HIP version differs from the official benchmark version of Laghos (in the
+top-level directory) in the following ways:
+
+1. Only problems 0 and 1 are defined
```

Comment:
Why only 0 and 1?

Reply:
It is as in all of the subdirectories there: the source `cuda` one has the same limitations.